### ISSUE TITLE
URDF with extension?

### ISSUE THREAD

**Issue by Tianchong-Jiang** [AlexanderKoch-Koch/low_cost_robot#32]
Hi Alex,

I built a follower arm and it was really nice. Thanks for the great work!

I wonder if you have an URDF file for the follower arm with the extension?  

I am also happy to add the extension to the URDF file myself. I've been editing the raw file and rendering the changes with rviz. It was difficult to align the axis of rotation and other stuff. What software have you been using to make the URDF file? 

Thank you!

Best regards,
Tianchong Jiang

![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4037546419103924667 skey=@crypt_877620b7_12ef54e85e45eddc67ac78fe2138bfe7 mmweb_appid=wx_webfilehelper](https://github.com/AlexanderKoch-Koch/low_cost_robot/assets/64213719/4d7ae2e5-f108-4d68-8599-2176ca1676fa)

**Comment by sinantie** [comment]
Hi Alex (and Tianchong),

This is an amazing project!
I have also built the follower and leader robots and tried to take a slightly different route. I have "updated" the Fusion follower_arm.f3z model (.f3z) instead with the elbow_to_wrist extension. I've been struggling a bit though to "export" to URDF... Trying to add the wrist extension within the MuJoCo visualiser by hand is quite hard, as it's a lot of guesswork to "nail" the `pos`. 

I've tried exporting with the Fusion to URDF plugin, but it doesn't work "out of the box"  which makes me think that it was not what was originally used (as the plugin requires to follow a few naming/design conventions which doesn't seem to be the case with the .f3z model).

Any thoughts? 

Thanks a lot!!

Cheers,

Yannis Konstas
